core/psh/Makefile

```diff
@@ -4,9 +4,9 @@
 # Copyright 2018, 2019, 2020, 2021 Phoenix Systems
 #
 
-OBJS = $(filter-out psh psh-empties, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh/ -name '*.c'))))
+OBJS = $(filter-out psh psh-empties login, $(subst .c,,$(subst ./core/psh/,, $(shell find ./core/psh/ -name '*.c'))))
 PSH_COMMANDS ?= $(OBJS)
-OBJ_FEATURED = $(addsuffix .o,$(filter $(PSH_COMMANDS),$(OBJS)))
+OBJ_FEATURED = $(addsuffix .o, $(if $(filter-out login,$(PSH_COMMANDS)) , $(PSH_COMMANDS) , $(if $(filter login,$(PSH_COMMANDS)),$(OBJS) login,$(OBJS)) ))
```

Comment:
i think login is not an applet, but compilation option of `psh` interactive shell, should not be treated as an applet.